### PR TITLE
Remove the alerting patch. Use ClickHouse built-in plugins backend instead.

### PR DESCRIPTION
**Pull request by gongled**
### What's this PR about

This PR removes the maintained patch that has provided alerting engine support for the ClickHouse data source in favour of the built-in plugin backend from Vertamedia.

Fortunately, it is not necessary to adapt the following patch every Grafana's upgrade. Now Grafana 7+ supports new backend API that is using to run 3rd-party server-side plugins (written in Go), not only on the user-side. 

So, we can nice and easy prune the implementation of the same functionality from the core forever. Is it great, isn't it?

### TODO's:

- [x] Run [`perfecto`](https://github.com/essentialkaos/perfecto) check on your spec file
- [x] Write [`bibop`](https://github.com/essentialkaos/bibop) tests
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

**Did you try to build the package with this spec?:** Yes
**Is this ready for review?:** No

### CODE REVIEW

File: specs/grafana-plugin-clickhouse-datasource.spec [hunk 1]
```diff
@@ -45,7 +45,7 @@
 Summary:              Clickhouse datasource for Grafana
 Name:                 grafana-plugin-clickhouse-datasource
 Version:              2.1.0
-Release:              0%{?dist}
+Release:              1%{?dist}
```
Comment (andyone) [inline specs/grafana-plugin-clickhouse-datasource.spec:48]: I think we should not bump the package(s) release. It still in the testing repo, so we can just replace the previously built version.

**Review by andyone — CHANGES_REQUESTED** [review]

**Comment by andyone** [comment]
@gongled Hi! Do you have any ETA for this PR?

**Comment by gongled** [comment]
@andyone sorry for delay. I've reverted `%{dist}` macro changes.

**Comment by andyone** [comment]
Ouch! Looks like the latest version of bibop is broken. I'll try to fix it ASAP. Pay no attention to build status.

File: specs/grafana/grafana.spec [hunk 2]
```diff
@@ -195,7 +193,10 @@ fi
 ################################################################################
 
 %changelog
-* Wed Sep 09 2020 Gleb Goncharov <g.goncharov@essentialkaos.com> - 7.1.5-0
+* Wed Sep 16 2020 Gleb Goncharov <g.goncharov@fun-box.ru> - 7.1.5-1
```
Comment (andyone) [inline specs/grafana/grafana.spec]: Changelog still contains info about release 1.

File: specs/grafana-plugin-clickhouse-datasource.spec (hunk 1 above)
Comment (andyone) [reply specs/grafana-plugin-clickhouse-datasource.spec:48]: This release bump also should be reveted.

**Review by andyone — CHANGES_REQUESTED** [review]

File: specs/grafana/grafana.spec (hunk 2 above)
Comment (gongled) [reply specs/grafana/grafana.spec]: Fixed (28e9f58)

File: specs/grafana-plugin-clickhouse-datasource.spec (hunk 1 above)
Comment (andyone) [reply specs/grafana-plugin-clickhouse-datasource.spec:48]: ☝️

**Review by andyone — CHANGES_REQUESTED** [review]

**Comment by andyone** [comment]
@gongled Hi! Is there any plans with this PR?

**Pull request closed without merging**